vision/Gemfile

```diff
@@ -1,4 +1,4 @@
-# Copyright 2016 Google, Inc
+# Copyright 2019 Google LLC
```

Comment:
Nit: For pre-existing files, you can leave the year and `Inc.` I think just the comma was wrong.